pupil-spa/src/app/check/check.component.ts

```diff
@@ -261,18 +263,10 @@ export class CheckComponent implements OnInit {
       }
       case CheckComponent.submissionPendingRe.test(stateDesc): {
         // Display pending screen
-        this.auditService.addEntry(new CheckSubmissionPending());
-        this.storageService.setItem('pending_submission', true);
+        this.storageService.setItem('preview_completed', true);
```

Comment:
The tests are ignored because there are no server calls now — there is no way to log in and take the test as normal; the normal flow is through the survey.

The pending submission screen is ‘replaced’ with `ict-survey/preview-completed`.